tracer/test/Datadog.Trace.Tests/TracerTests.cs

```diff
@@ -524,5 +524,48 @@ public async Task ForceFlush()
 
             agent.Verify(a => a.FlushTracesAsync(), Times.Once);
         }
+
+        [Fact]
+        public void SetUser()
+        {
+            var scopeManagerMock = new Mock<IScopeManager>();
+
+            var spanContext = new SpanContext(1ul, 2ul);
+            var span = new Span(spanContext, DateTime.UtcNow);
+            var scope = new Scope(null, span, scopeManagerMock.Object, true);
+            scopeManagerMock.Setup(x => x.Activate(It.IsAny<Span>(), It.IsAny<bool>())).Returns(scope);
+            scopeManagerMock.SetupGet(x => x.Active).Returns(scope);
+
+            var settings = new TracerSettings
+            {
+                StartupDiagnosticLogEnabled = false
+            };
+
+            var tracer = new Tracer(settings, Mock.Of<IAgentWriter>(), Mock.Of<ISampler>(), scopeManagerMock.Object, Mock.Of<IDogStatsd>());
+
+            var testScope = tracer.StartActive("test.trace");
```

Comment:
Should we add a child span and assert the tags were _not_ set on the child?

Reply:
Good idea, I've added this to 87a45264a.